Woof/Woof/Model/Sitter/Sitter.swift

```diff
@@ -29,4 +29,62 @@ struct Sitter: Codable, Identifiable {
 
     /// The price per hour charged by the sitter.
     var pricePerHour: Double
+
+    /**
+     Creates a pet sitter instance with the specified parameters with the unique identifier.
+
+     - Parameters:
+     - name: The first name of the pet sitter.
+     - surname: The last name of the pet sitter.
+     - phone: The phone number of the pet sitter.
+     - avatarURL: The URL that provides an avatar image of the pet sitter.
+     - bio: The additional information about the pet sitter.
+     - rating: The rating of the pet sitter.
+     - pricePerHour: The price per hour charged by the pet sitter.
+
+     Each instance of the new pet sitter has a unique id.
```

Comment:
This line should be before the parameters.

Reply:
How does it not the part PR, if you added this information?

Reply:
You didn't fix the comment.

Reply:
``` swift
 /**
     Creates a pet sitter instance with the specified parameters with the unique identifier.
     Each instance of the new pet sitter has a unique id.

     - Parameters:
      - name: The first name of the pet sitter.
      - surname: The last name of the pet sitter.
      - phone: The phone number of the pet sitter.
      - avatarURL: The URL that provides an avatar image of the pet sitter.
      - bio: The additional information about the pet sitter.
      - rating: The rating of the pet sitter.
      - pricePerHour: The price per hour charged by the pet sitter.
     */
```

Reply:
Line 45 now is before the parameters. Or you've meant smh else?